aspnetcore/security/authentication/identity.md

```diff
@@ -12,12 +12,24 @@ uid: security/authentication/identity
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-ASP.NET Core Identity is a membership system that supports user interface (UI) login functionality. Users can create an account with the login information stored in Identity or they can use an external login provider. Supported external login providers include [Facebook, Google, Microsoft Account, and Twitter](xref:security/authentication/social/index).
+ASP.NET Core Identity:
 
-Identity can be configured using a SQL Server database to store user names, passwords, and profile data. Alternatively, another persistent store can be used, for example, Azure Table Storage.
+* Is a database API that supports user interface (UI) login functionality.
```

Comment:
Database isn't the right word here, since identity is supposed to be store implementation agnostic, maybe just drop the database word, or Storage might be better?  Ideas on wording? @blowdart @ajcvickers

Reply:
I deleted Database - later on I say
Identity is typically configured using a SQL Server database to store user names, passwords, and profile data. Alternatively, another persistent store can be used, for example, Azure Table Storage.